.gitignore

```diff
@@ -9,5 +9,5 @@
 # These are backup files generated by rustfmt
 **/*.rs.bk
 
-# Generated by VS Code
-**/.vscode
+# Files from macOS
+.DS_Store
```

Comment:
I would actually like to continue to ignore `.vscode/` directories _by default_. This will require using `-f` or `--force` with `git add` to make changes to the VS Code settings we do want, but I expect that to be exceptionally rare.

```suggestion
# Generated by VS Code
**/.vscode
# Files from macOS
.DS_Store
```